### ISSUE TITLE
Replace new_features.rst

### ISSUE THREAD

**Issue by josephj11** [autokey/autokey#680]
### Has this issue already been reported?

- [X] I have searched through the existing issues.

### Is this a question rather than an issue?

- [X] This is not a question.

### What type of issue is this?

Documentation

### Which Linux distribution did you use?

N/A

### Which AutoKey version did you use?

0.96.0 Beta 10

### How did you install AutoKey?

N/A

### Can you briefly describe the issue?

New_features.rst is around a decade old and was never very helpful.

### Anything else?

This needs to either be rewritten and made prominent or just deleted.

We do have a number of new features in 0.96 that need to be highlighted somewhere.

- Mouse dragging/moving
- Get trigger abbreviation and character
- Default to pasting with Ctrl+v
- Keyboard.send_keys() output method parameter
- New format for API documentation
- Probably more

**Comment by Elliria** [comment]
I thought I'd add links to the **new_features.rst** file on the [develop](https://github.com/autokey/autokey/blob/develop/new_features.rst) and [master](https://github.com/autokey/autokey/blob/master/new_features.rst) branches here.

**Comment by josephj11** [comment]
This is where visgrep() was documented and probably why no one ever used it - because no one read it. So don't get your hopes up about anyithing you puti here.

**Comment by Elliria** [comment]
Yep. I just added to the issue in passing as I was relabeling it. I also added it to the [0.97.0 milestone](https://github.com/autokey/autokey/milestone/2) so that we don't overlook it when preparing for that release.
